chapters/syntax.tex

```diff
@@ -63,10 +63,7 @@ \section{Lexical conventions}\label{lexical-conventions}
   (Previously the grammar used underscore.)
 \end{itemize}
 
-\begin{nonnormative}
-Within a description-string the optional tags \lstinline!<HTML>! and \lstinline!</HTML>! or \lstinline!<html>! and \lstinline!</html>! define the start and end of content that is HTML encoded.
-\end{nonnormative}
-
+If a description-string starts with the tag \lstinline!<html>! or \lstinline!<HTML>! the entire string is HTML encoded (and is assumed to end with \lstinline!</html>! or \lstinline!</HTML>! and shall be rendered as HTML even if the end-tags are missing), otherwise the entire string is rendered as is.
```

Comment:
Slightly easier to parse:
```suggestion
If a description-string starts with the tag \lstinline!<html>! or \lstinline!<HTML>!, the entire string is HTML encoded (assumed to end with \lstinline!</html>! or \lstinline!</HTML>!, but to be rendered as HTML even if the end-tag is missing).
Otherwise, the entire string is rendered as is.
```

Reply:
I'm fine with that change as well.

Reply:
I used the same text as we already have for `Documentation`:

https://github.com/modelica/ModelicaSpecification/blob/6eeb87f67c83327b8321740826a057b539577f37/chapters/annotations.tex#L166

I think we should have the same language, so I would prefer keeping the PR as is.

Reply:
If that's more convenient, I can make a separate follow-up PR to consistently improve readability in both places.